### ISSUE TITLE
Finetuning for 3 class segmentation

### ISSUE THREAD

**Issue by salohiddin22** [rekalantar/MedSegmentAnything_SAM#1]
Hi Reza. Thanks for the detailed explanation in your blog and the code.

I was trying to use your code for 3 class segmentation on the whole image. For the binary it worked fine.

I mainly changed the following parts.
- The bounding box to the image size. 
- ScaleIntensityRanged(keys=['label'], a_min=53, a_max=226, b_min=0.0, b_max=2.0, clip=True),
- seg_loss = monai.losses.DiceCELoss(to_onehot_y=True, softmax=True, squared_pred=True, reduction='mean')". 

But I cannot get any results it's just giving black image as prediction.

Can you suggest anything to fit it for multi-class segmentation?

Many thanks!
![image](https://github.com/rekalantar/MedSegmentAnything_SAM_LungCT/assets/89576437/516b91bd-03b4-49cf-9eb8-714dc8501f76)
